overrides/offline.yaml

```
@@ -1,4 +1,4 @@
-os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }}_{{ ansible_distribution|lower }}_{{ ansible_distribution_major_version }}_x86_64.tar.gz"
+os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }}_{{ ansible_distribution|lower }}_{{ ansible_distribution_version }}_x86_64.tar.gz"
```

Since we are making a change here maybe this is a good opportunity to fix the formatting to make it match the other artifacts? 
```suggestion
os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }} -{{ ansible_distribution|lower }}-{{ ansible_distribution_version }}_x86_64.tar.gz"
```

---

i don't know about this i would really not touch this right now.